libpromises/evalfunction.c

```diff
@@ -3152,6 +3153,69 @@ static FnCallResult FnCallFileStatDetails(ARG_UNUSED EvalContext *ctx,
     {
         return FnFailure();
     }
+    else if (!strcmp(detail, "xattr"))
+    {
+#if defined(WITH_XATTR)
+        // Extended attributes include both POSIX ACLs and SELinux contexts.
+        char attr_raw_names[CF_BUFSIZE];
+        ssize_t attr_raw_names_size = listxattr(path, attr_raw_names, sizeof(attr_raw_names));
+
+        if (attr_raw_names_size < 0)
+        {
+            if (errno != ENOTSUP && errno != ENODATA)
+            {
+                Log(LOG_LEVEL_INFO, "Can't read extended attributes of '%s'. (listxattr: %s)",
```

Comment:
OK.  I don't think of a system probe error as an actual error, that's why I had `INFO` before.